recipes/sdl2_mixer/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# Make sure TravisCI can find SDL2
+if [ `uname` == Darwin ]; then
+	export LDFLAGS="${LDFLAGS} -Wl,-rpath,$PREFIX/lib"
```

Comment:
Could you please add a PR to the [`toolchain`]( https://github.com/conda-forge/toolchain-feedstock ) to include this?